content/code-signing-yaml/signing-macos.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Signing macOS apps
+description: How to set up macOS code signing in codemagic.yaml
+weight: 2
+aliases: '../code-signing-yaml/signing'
+---
+
+All macOS applications have to be digitally signed before they can be installed on devices or made available to the public via Mac App Store or outside of Mac App Store.
+
+{{<notebox>}}
+This guide only applies to workflows configured with the **codemagic.yaml**. If your workflow is configured with **Flutter workflow editor** please go to [Signing macOS apps using the Flutter workflow editor](../code-signing/ios-code-signing).
```

Comment:
Link looks incorrect here, it should point to mac os codesigning.

Reply:
overlooked, fixed in https://github.com/codemagic-ci-cd/codemagic-docs/pull/320/commits/631fb2e517b63ef00d3ef2a2d6a09183576fcca4#diff-ed26953a6de599789eded1bb99c9cbe95db2ca812b27a37ae43c5a8499c1a38dR10